.github/workflows/release/js-pre-npm-build.sh

```diff
@@ -0,0 +1,10 @@
+echo "Build Chat SDK module artifacts"
+pushd ..
+./gradlew :jsNodeProductionLibraryDistribution
```

Comment:
Is jsNodeProductionLibraryDistribution task defined somewhere in our code?

Reply:
it's a built in kotlin task